cli/src/api/mod.rs

```diff
@@ -573,6 +589,66 @@ impl PhylumApi {
     }
 }
 
+/// Phylum group types.
+pub enum Group {
+    Legacy(String),
+    Org(OrgGroup),
+}
+
+impl Group {
+    /// Create a new group.
```

Comment:
This function doesn't create a new group. It creates a `Group` value referencing an organization group.

Reply:
I don't think this is particularly misleading on a constructor. It does create a new `Group`, that's exactly what a constructor does.

Reply:
It creates a new `Group`, but not a new group, which is implemented by a different function.

Reply:
I don't think that would surprise anyone? This function is not taking an API argument? I can remove the doc comment but I don't really see the issue.